app/angular/src/server/__tests__/angular-cli_config.test.ts

```diff
@@ -1,5 +1,5 @@
-import { getAngularCliWebpackConfigOptions } from './angular-cli_config';
-import angularJson from '../../../../examples/angular-cli/angular.json';
+import { getAngularCliWebpackConfigOptions } from '../angular-cli_config';
```

Comment:
just checking what is happening around, do we have some new convention to put tests in the `__tests` dir ?

Reply:
We don't. I just wanted to bring some order in the server directory. 

![image](https://user-images.githubusercontent.com/5478899/58023091-3aa30980-7b0f-11e9-8147-74e77543ab21.png)

Is it worth to caring about this? I don't know. I can leave it as is 🙂

Reply:
I like colocation of tests, until the folder gets like 6+ files, then having a `tests` folder is kinda nice. I don't care enough about it either way really. 👍